src/SSHDebugPS/StringResources.resx

```diff
@@ -162,6 +162,9 @@
     <value>Source file '{0}' not found.</value>
     <comment>{0} = source file name</comment>
   </data>
+  <data name="Error_DockerTypesNotMatching" xml:space="preserve">
+    <value>Make sure your Docker Desktop container type matches the selected Connection type.</value>
```

Comment:
Change message to say `Ensure the target Docker container target type is Linux.` You know that for this connection it has to be a Linux connect.